Automattic/wp-calypso
# chore(packages): [@automattic/social-previews] Add missing dev dependencies

#### Changes proposed in this Pull Request

Adds missing dependencies. It should fix

```
➤ YN0000: ┌ /Users/sergio/src/automattic/wp-calypso/packages/social-previews/package.json
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/packages/social-previews/package.json:51:32: Unmet transitive peer dependency on jest@>=26.4.0, via @automattic/calypso-build@^7.0.0
➤ YN0000: │ /Users/sergio/src/automattic/wp-calypso/packages/social-previews/package.json:51:32: Unmet transitive peer dependency on webpack@^5.13.0, via @automattic/calypso-build@^7.0.0
➤ YN0000: └ Completed in 0s 351ms
```

#### Testing instructions

* Run `npx @yarnpkg/doctor packages/social-previews` and verify the above error is gone
